### PR TITLE
Wrong name for Header of Solution box

### LINKED ISSUE #13: Solution APE03E3
I can take care of this one too! :)

### PR DESCRIPTION
**Pull request by gauthmug**
A grey bar is added around a solution. At the top of this grey bar, a
header indicates “Solution to APE 2”, if we are looking at Exercise 2.
The header should be “Solution to Exercise 2”, instead of “Solution to
APE 2”.

PS. I believe there are some problems about the numbering of the exercise sessions posted on Moodle. For example, the file “APE9-Bargaining-2017-04-18.pdf” posted on Moodle is such that the title of this file is “Session 10 – Bargaining and Cooperation” which is probably wrong :p

Closes #13 #20 #21 #54 #58 #59

### CODE REVIEW

File: notes/Chapter-Auctions/chap7.tex [hunk 1]
```diff
@@ -64,7 +64,7 @@ \section{Definitions and Assumption}
...
+Note further that agents with such utility functions are \emph{risk neutral}. Suppose that an agent must decide whether she wants to participate in a fair lottery, that awards the amount $x$ half of the time and the amount $-x$ the other half of the time. The expected utility is zero--- thus the agent is indifferent between participating in the lottery or not participating. In real live the situation is often more complicated, as for instance \emph{risk seeking} agents might prefer to participate in a lottery that cost \$\,$1$ and has a 0.01 \% chance of paying off \$\,$1000$, whereas a risk neutral player is indifferent.
```
Comment (blegat) [inline notes/Chapter-Auctions/chap7.tex]: Why this change ? 0.01 * 1000 = 10 and 0.001 * 1000 = 1 right ?

Comment (gauthmug) [inline notes/Chapter-Auctions/chap7.tex]: Yes indeed, then I would remove the % sign ;)

Comment (gauthmug) [inline notes/Chapter-Auctions/chap7.tex]: BTW I don't manage to create a new Pull Request, all my commits arrive here, I am sorry about that :(

**Comment by gauthmug** [comment]
There is a typo in the title of commit https://github.com/UCL-INMA/LINMA2345/pull/53/commits/c1bc345af703f4e763d81b548d54899a5b2b11fc.
I wrote

> Improved solution for APE03E03


but it should be

> Improved solution for APE03E01

Sorry about that!

File: notes/Chapter-Auctions/chap7.tex (hunk 1 above)
Comment (blegat) [inline notes/Chapter-Auctions/chap7.tex]: It should be 0.1 % then, shouldn't it ?

Comment (gauthmug) [inline notes/Chapter-Auctions/chap7.tex]: Indeed, it should be

> a 0.1 \% chance

or 

> a 0.001 chance


Thanks a lot! :)

Comment (gauthmug) [inline notes/Chapter-Auctions/chap7.tex]: How do I create a new branch?

Comment (gauthmug) [inline notes/Chapter-Auctions/chap7.tex]: @blegat I am afraid you are going to get crazy with this Pull Request because there are so many files changed in it :o

Comment (blegat) [inline notes/Chapter-Auctions/chap7.tex]: `git branch branchname` creates a branch
`git checkout branchname` goes to the branch

Comment (blegat) [inline notes/Chapter-Auctions/chap7.tex]: https://git-scm.com/book/fr/v2/Les-branches-avec-Git-Les-branches-en-bref

File: exercises/APE2/0201.tex [hunk 2]
```diff
@@ -153,41 +153,150 @@
...
 
 	      \end{itemize}
+				
+				The payoff function can be represented by these two tables.
+			 
+			 \begin{center}
+				 \begin{tabular}{l|cc}
+					 		high costs & enter & not enter \\
+					 		\hline
+					 		open        & -10/2 & -4/0 \\
```
Comment (blegat) [inline exercises/APE2/0201.tex]: -10 -> $-10$

File: notes/Chapter-Auctions/chap7.tex [hunk 3]
```diff
@@ -276,7 +286,7 @@ \section{Double auctions}
    p_b(v_b) &= \begin{cases} x, &\text{if } v_b \geq x \\ 0, &\text{otherwise}
    \end{cases}
    & 
-   p_s(v_s) &= \begin{cases} x, &\text{if } v_s \leq x \\ 0, &\text{otherwise} \end{cases}
+   p_s(v_s) &= \begin{cases} x, &\text{if } v_s \leq x \\ 1, &\text{otherwise} \end{cases}
```
Comment (blegat) [inline notes/Chapter-Auctions/chap7.tex]: Good catch !

File: notes/Chapter-Auctions/chap7.tex (hunk 1 above)
Comment (blegat) [inline notes/Chapter-Auctions/chap7.tex]: I went through it, it fixes 6 issues at the same time but I managed to stay sane :-P

Comment (gauthmug) [inline notes/Chapter-Auctions/chap7.tex]: Okay, I don't think I will have the time to master GitHub in a couple of hours, but it is fun to use it! Furthermore, I don't think I will contribute a lot between now and the exam since I will focus on the theory now :)
However, I have a new problem, this time with the interface GitHub Desktop: I have created a Pull Request here: https://github.com/gauthmug/LINMA2345/pull/1
Obviously, it shouldn't be there, and I am not even sure you can see what's there :'(

Comment (blegat) [inline notes/Chapter-Auctions/chap7.tex]: This pull request tries to add changes of this repo (i.e. `upstream`) to your fork. Typically, this is done instead by you doing
```
$ git pull upstream master
$ git push origin master
```
without going through pull requests

Comment (gauthmug) [inline notes/Chapter-Auctions/chap7.tex]: Okay I may have found a solution.
Can you see https://github.com/UCL-INMA/LINMA2345/pull/60/commits/c2795d9a26af97b9dd7c35c07746a7a7f96c3f09 and https://github.com/UCL-INMA/LINMA2345/pull/60/commits/7980c98c49bdf39c1cb6bdec54dad37e89e9cc04 ?

Comment (blegat) [inline notes/Chapter-Auctions/chap7.tex]: A solution to what ?

Comment (gauthmug) [inline notes/Chapter-Auctions/chap7.tex]: My problem was that I was creating a pull request to my private repository
Now the problem is gone, more or less, I think :p

Comment (gauthmug) [inline notes/Chapter-Auctions/chap7.tex]: Can you see https://github.com/UCL-INMA/LINMA2345/pull/60/commits/7980c98c49bdf39c1cb6bdec54dad37e89e9cc04 ?

Comment (blegat) [inline notes/Chapter-Auctions/chap7.tex]: Yes
